.github/workflows/issue-first-response.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 New Relic Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: issue-first-response
+
+on:
+    issues:
+        types:
+            - opened
```

Comment:
I don't think all opened issues require this type of response. Some issues are enhancement requests to add support for a framework or a new PHP version. I think this should be changed to a 'label' if the issue is determined to be a problem report rather than 'enhancement' request.

Reply:
we can add the logic to only give this response to "bugs"
`if: github.event.label.name == 'bug' && github.event.issue.state == 'open'`

Can work with PM if he'd like specific text for the feature enhancements.

Reply:
Implemented here 6e48075d7014361aacf105c55c734481bc429198